cert/lib/cert/options.rb

```diff
@@ -100,9 +100,11 @@ def self.available_options
                                      env_name: "CERT_KEYCHAIN_PATH",
                                      description: "Path to a custom keychain",
                                      code_gen_sensitive: true,
-                                     default_value: Dir["#{Dir.home}/Library/Keychains/login.keychain", "#{Dir.home}/Library/Keychains/login.keychain-db"].last,
+                                     default_value: Helper.mac? ? Dir["#{Dir.home}/Library/Keychains/login.keychain", "#{Dir.home}/Library/Keychains/login.keychain-db"].last : nil,
```

Comment:
Modified as default_value is also a subject for `verify_block` and will throw errors

Reply:
What's the meaning of `[].last` ?

Reply:
Honestly, Im not sure, I tried not to touch to avoid accidental behavior changes. Seems like first added [here](https://github.com/fastlane/fastlane/commit/cefd4ee1e2c04fc87b00d94a17b0718dc27516ec) and then modified [here](https://github.com/fastlane/fastlane/pull/6928/files)